### PR TITLE
Create sfo.md

### LINKED ISSUE #97: manjeet88
## Workflow steps
- [x] Create a new branch
- [x] Add commits
- [x] Open a pull request
- [x] Discuss and review your code
- [x] Merge :ship: :tada:

### PR DESCRIPTION
**Pull request by manjeet88**
test

closes #97 

fixes #97 

resolves #97

### CODE REVIEW

File: sfo.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#sfo, ca
```
Comment (lucy-sheridan) [inline sfo.md:1]: more text

Comment (manjeet88) [inline sfo.md:1]: @lucy-sheridan  - Added :+1:

**Comment by manjeet88** [comment]
This is fine :+1:

**Comment by manjeet88** [comment]
Squashing the comment

**Pull request merged**